src/Language/PureScript/Parser/Lexer.hs

```diff
@@ -185,10 +185,15 @@ parsePositionedToken = P.try $ do
 parseToken :: P.Parsec String u Token
 parseToken = P.choice
   [ P.try $ P.string "<-" *> P.notFollowedBy symbolChar *> pure LArrow
+  , P.try $ P.string "←"  *> P.notFollowedBy symbolChar *> pure LArrow
```

Comment:
Yes, since we added unicode operators. @DavidLindbom could you please just verify that something like `←→` can be used as an operator (ideally in a test)?